packages/core/utils/src/serializer.js

```diff
@@ -1,5 +1,21 @@
 // @flow
 
+const nameToCtor: Map<string, Class<*>> = new Map();
+const ctorToName: Map<Class<*>, string> = new Map();
```

Comment:
Currently these cannot be cleared. Since they don't grow over time, and once constructors and require cache entries are loaded into memory, they can't be unloaded, I don't think being able to dispose of this would be very impactful.

Reply:
doesn't seem like a big deal. there are only a few of them